.github/workflows/README.md

```diff
@@ -0,0 +1,66 @@
+<!-- Note: This document cannot be in `.github/README.md` because it will overwrite the repo README.md -->
+
+# Continuous Integration and Delivery
+
+## Unit and Integration Testing
+
+| workflow name               | workflow file                       | action                                                                                                                                                         | accelerator\* | (Python, PyTorch)                                | OS                  |
+| --------------------------- | ----------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | ------------------------------------------------ | ------------------- |
+| Test full                   | .github/workflows/ci_test-full.yml  | Run all tests except for accelerator-specific, standalone and slow tests.                                                                                      | CPU           | (3.7, 1.8), (3.7, 1.11), (3.9, 1.8), (3.9, 1.11) | linux, mac, windows |
+| Test with Conda             | .github/workflows/ci_test-conda.yml | Same as ci_test-full.yml but with dependencies installed with conda.                                                                                           | CPU           | (3.8, 1.8), (3.8, 1.9), (3.8, 1.10), (3.9, 1.11) | linux               |
+| Test slow                   | .github/workflows/ci_test-slow.yml  | Run only slow tests. Slow tests usually need to spawn threads and cannot be speed up or simplified.                                                            | CPU           | (3.7, 1.8)                                       | linux, mac, windows |
+| PL.pytorch-lightning (IPUs) | .azure-pipelines/ipu-tests.yml      | Run only IPU-specific tests.                                                                                                                                   | IPU           | (3.8, 1.9)                                       | linux               |
+| PL.pytorch-lightning (HPUs) | .azure-pipelines/hpu-tests.yml      | Run only HPU-specific tests.                                                                                                                                   | HPU           | (3.8, 1.10)                                      | linux               |
+| PL.pytorch-lightning (GPUs) | .azure-pipelines/gpu-tests.yml      | Run all CPU and GPU-specific tests, standalone and examples. Each standalone test needs to run individually to avoid unwanted interactions between test cases. | GPU           | (3.7, 1.8)                                       | linux               |
```

Comment:
```suggestion
| PL.pytorch-lightning (GPUs) | .azure-pipelines/gpu-tests.yml      | Run all CPU and GPU-specific tests, standalone and examples. Each standalone test needs to be run in separate processes to avoid unwanted interactions between test cases. | GPU           | (3.7, 1.8)                                       | linux               |
```